### PR TITLE
ME AI Sentiment Analysis CSharp

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**

### CODE REVIEW

File: samples/msgext-ai-sentiment-analysis/csharp/MEAISentimentAnalysis/Pages/SentimentModel.cshtml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+﻿@page
+@model MEAISentimentAnalysis.Pages.SentimentModel
+@{
+}
+<head>
+    <script src="https://res.cdn.office.net/teams-js/2.14.0/js/MicrosoftTeams.min.js"
```
Comment (Pawank-MSFT) [inline samples/msgext-ai-sentiment-analysis/csharp/MEAISentimentAnalysis/Pages/SentimentModel.cshtml:6]: User Latest SDK version, Instead of 2.14.0

**Review by Pawank-MSFT — APPROVED** [review]
Just update latest version , and other comments.

File: samples/msgext-ai-sentiment-analysis/csharp/MEAISentimentAnalysis/Pages/SentimentModel.cshtml (hunk 1 above)
Comment (Mohammed-MSFT) [reply samples/msgext-ai-sentiment-analysis/csharp/MEAISentimentAnalysis/Pages/SentimentModel.cshtml]: Updated

File: samples/msgext-ai-sentiment-analysis/csharp/assets/sample.json [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      {
+        "key": "TEAMS-FEATURES",
+        "value": "ME, Azure Open AI"
+      }
+    ],
+    "thumbnails": [
+      {
+        "type": "image",
+        "order": 100,
+        "url": "https://raw.githubusercontent.com/OfficeDev/Microsoft-Teams-Samples/main/samples/msgext-ai-sentiment-analysis/csharp/Images/Sentiment_Analysis",
```
[the remark this answers is not in the document]
Comment (Pawank-MSFT) [reply samples/msgext-ai-sentiment-analysis/csharp/assets/sample.json]: Fixed

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Apporving!

**Review by Prithvi-MSFT — APPROVED** [review]

**Comment by Pawank-MSFT** [comment]
Executed build pipeline:
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6766481308

**Review by Wajeed-msft — APPROVED** [review]

**Comment by Pawank-MSFT** [comment]
Execute builld pipeline: 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6979064312

**Pull request merged**